### ISSUE TITLE
Add to cart button on product page should announce that an item was added to cart

### ISSUE THREAD

**Issue by huguestennier** [netlify/swag-site#35]
Currently, if you use a keyboard & a screen reader and add a product to your cart, nothing is announced.

We should include a live region that gets updated every time you add an item to your cart announcing something like:

`[quantity] [product title] added to your cart`

**Comment by huguestennier** [comment]
Closed via https://github.com/netlify/swag-site/pull/70
